src/index.mjs

```diff
@@ -29,6 +30,18 @@ export { ReactNativeFile } from './ReactNativeFile'
  * ```
  */
 
+/**
+ * @kind typedef
```

Comment:
@jaydenseric Is this the correct way for declaring the type?

Reply:
I added a bit of polish to the JSDoc :) This one needed a description.